stl/inc/vector

```diff
@@ -1362,20 +1503,40 @@ public:
     template <class _Iter, enable_if_t<_Is_iterator_v<_Iter>, int> = 0>
     _CONSTEXPR20 void assign(_Iter _First, _Iter _Last) {
         _Adl_verify_range(_First, _Last);
+        auto _UFirst = _Get_unwrapped(_First);
+        auto _ULast  = _Get_unwrapped(_Last);
+        if constexpr (_Is_fwd_iter_v<_Iter>) {
+            using _SizeTy      = make_unsigned_t<typename iterator_traits<_Iter>::difference_type>;
+            const auto _Length = static_cast<_SizeTy>(_STD distance(_UFirst, _ULast));

```

Comment:
Why does this introduce `_SizeTy` while other places use `size_t`?

https://github.com/microsoft/STL/blob/3e6b68fccb9db66d8d7dbfc20688ff7247078b52/stl/inc/vector#L698

Reply:
No idea - this doesn't seem at all justifiable.